src/main/resources/db/migration/V52__migrer_vedtak_uten_saksbehandler_og_beslutter_ident.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE vedtak
+SET saksbehandler_ident = subquery.opprettet_av
+FROM (SELECT opprettet_av, behandling_id FROM behandlingshistorikk WHERE steg = 'SEND_TIL_BESLUTTER') as subquery
+WHERE vedtak.behandling_id = subquery.behandling_id;
+
+UPDATE vedtak
+SET beslutter_ident = subquery.opprettet_av
+FROM (SELECT opprettet_av, behandling_id FROM behandlingshistorikk WHERE steg = 'BESLUTTE_VEDTAK') as subquery
```

Comment:
Finnes det behandlingshistorikk for dette steget? Eller er det her vi ikke lagrer ned historikken? 🤔

Reply:
Finnes. Usikker på hvor raden lages, men det dukker opp i tabellen når man kjører lokalt.

Reply:
Interessant - `override fun settInnHistorikk(): Boolean = false` i `BeslutteVedtakSteg`

Reply:
Ok, my bad;
```
behandlingshistorikkService.opprettHistorikkInnslag(behandling = behandling,
                                                            utfall = utfall,
                                                            metadata = beslutteVedtak)

```